libmscore/read301.cpp

```diff
@@ -289,6 +291,27 @@ bool Score::read(XmlReader& e)
       return true;
       }
 
+//---------------------------------------------------------
+// linkMeasures
+//---------------------------------------------------------
+
+void Score::linkMeasures(Score* score)
+      {
+      MeasureBase *mbMaster = score->first();
```

Comment:
I think MeasureBaseList is a good candidate for an implementation of this method

So here you can leave something like that:
`if (!score || _measure.size() == 0)
      return;

_measures->link(score->measures());
`

Also I'd suggest to add a very simple method to MeasureBaseList - "bool hasNext() const;"
It'll slightly improve your loops.